ADA/ada.qmd

```diff
@@ -19,57 +17,200 @@ It brings together:
+---
+
+#### Getting access to your data
+
+---
+
+Data in Databricks is held in catalogs in the Delta Lake, which is centrally governed by the ADA team. Catalogs provide different levels of access control and security. In Databricks, databases are referred to as "schemas", with tables sitting below them. You can find out more information about this on the [ADA guidance site](https://analytical-data-access.azurewebsites.net/get-support/how-databricks-works). 
+
+You will need to be given access to Databricks and the data you require by the ADA team by completing an "Analytical Data Access Request" service request form in the IT Help Centre. At the moment, not all of the data currently available in SSMS has been migrated to the Delta Lake for use in Databricks. You can [check the status of various datasets](https://app.powerbi.com/groups/me/reports/966389db-2e4e-42b6-8416-36aee2fb9503/ReportSection?ctid=fad277c9-c60a-4da1-b5f3-b3b8b34a82f9&experience=power-bi) in the data catalogue dashboard. 
+
+To access your data and make use of cloud computing resource, you will need access to a cluster, which is a group of virtual machines that complete your processing for you. Your cluster setup will depend on which programming language you wish to use. You will be given access to a shared cluster when your data access is granted. Please see the [how clusters work](https://analytical-data-access.education.gov.uk/get-support/how-clusters-work) page on the ADA guidance site for more detail. 
+
+---
+
+#### Writing and running code
+
+---
+
+In the Databricks interface, you have the choice to use notebooks, scripts or the Databricks SQL Editor. 
+
+Notebooks are made up of multiple code or markdown cells. They are different to IDEs (Integrated Development Environments) like RStudio. You can read more about the differences between IDEs and notebooks and where each might be useful [on the NHS-R website](/ADA/ada.html#differences-between-ides-and-notebooks).
+
+Scripts (e.g. R scripts) can be created and saved in your Workspace and run via the Databricks interface. 
+
+There is detailed guidance on using the Databricks SQL editor, which is similar to SSMS, on the [ADA guidance site](https://analytical-data-access.education.gov.uk/get-support/using-sql-in-databricks-sql-editor).
+
+One benefit of Databricks is the use of workflows to run your code. Workflows allow you to automate the running of single or multiple scripts, so if you have large amounts of data processing to do on a regular basis, you can set up a workflow to complete this overnight. Your laptop does not need to be switched on for this to work. 
+
+You can also run code from RStudio or other IDEs locally and execute it against the data held in Databricks. To do this, you will need to set up a connection between RStudio and the Databricks SQL Warehouse. This is explained in the [R scripts in RStudio section](/ADA/ada.html#r-scripts-in-rstudio) below.
+
+---
+
+#### Accessing code from a git repository
```

Comment:
Git should be a captial G I believe?

Reply:
Did you make an issue for this @jen-machin?

Reply:
I did not but thank you for reminding me

Reply:
Done!